### PR TITLE
chore: bump node-abi version to 3.0.0

### PR DESCRIPTION
**Pull request by xueqingxiao**
Closes https://github.com/prebuild/prebuild/issues/281

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,19 +2,9 @@ language: node_js
...
     os: linux
   - node_js: '12'
-    os: osx
+    os: linux
   - node_js: '10'
-    os: osx
-  - node_js: '8'
-    os: osx
-  - node_js: '6'
-    os: osx
```
Comment (ralphtheninja) [inline .travis.yml:20]: Why are you removing `osx`?

**Review by ralphtheninja — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (xueqingxiao) [reply .travis.yml:20]: my bad, I will revert osx support.

**Comment by ralphtheninja** [comment]
It's probably easier to get this PR through if you just bumped the dependencies. Sorry, but to avoid other discussions of removing support for older node.js versions etc.

**Comment by ralphtheninja** [comment]
@vweevers Do we bump major version when no longer supporting older node versions? I forget.

**Comment by mafintosh** [comment]
My personal take on node support + semver is that I normally major bump it unless the node version is very old at this stage, which I'd say 10 is - aka semverish 🤣

**Comment by xueqingxiao** [comment]
@ralphtheninja it's looks good for me, but we should find out that is the better way to resolve broken build with electron 14. which is related to #281.

**Comment by ralphtheninja** [comment]
> @ralphtheninja it's looks good for me, but we should find out that is the better way to resolve broken build with electron 14. which is related to #281.

I understand. If you just updated the dependencies we could release a patch pretty quickly, rather than having to debate removing node versions, which isn't related to what you really want to achieve. And we can postpone that for later. Makes sense?

**Comment by xueqingxiao** [comment]
> And
yep, but node-abi@3.0.0 is required minimal node v10 that is why I'm updated node engine.

**Comment by xueqingxiao** [comment]
> > @ralphtheninja it's looks good for me, but we should find out that is the better way to resolve broken build with electron 14. which is related to #281.
> 
> 
> 
> I understand. If you just updated the dependencies we could release a patch pretty quickly, rather than having to debate removing node versions, which isn't related to what you really want to achieve. And we can postpone that for later. Makes sense?

yep, but node-abi@3.0.0 is required minimal node v10 that is why I'm updated node engine.

**Comment by ralphtheninja** [comment]
> yep, but node-abi@3.0.0 required minimal node version is v10 that is why I'm updated node engine. you can see release note at [here](https://github.com/electron/node-abi/releases/tag/v3.0.0).

Fair enough. It wasn't really clear from the completely empty pull request message :wink:

**Review by ralphtheninja — APPROVED** [review]

**Comment by vweevers** [comment]
> Do we bump major version when no longer supporting older node versions? I forget.

IMO yes. We can't see on which node versions prebuild is used. What we do know is that dropping support can be a painful breaking change for consumers. In comparison, for folks on recent node, a semver-major signal is noise but not painful.

**Review by vweevers — APPROVED** [review]

**Comment by xueqingxiao** [comment]
@ralphtheninja @vweevers could you guys kindly merge the pr and release a new version?

**Comment by ralphtheninja** [comment]
`11.0.0`

**Comment by xueqingxiao** [comment]
Thanks a lot.

**Pull request merged**